### PR TITLE
Django 3.0 support

### PR DESCRIPTION
**Pull request by akinmetin**
Added Django 3.0 support by removing ``from django.utils import six`` and adding ``six``
Updated package version.

### CODE REVIEW

File: classytags/__init__.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.9.0'
+__version__ = '1.0.0'
```
Comment (yakky) [inline classytags/__init__.py]: Avoid bumping version in a PR

Comment (akinmetin) [reply classytags/__init__.py]: Reverted version to ```0.9.0``

**Comment by yakky** [comment]
Thanks, I merged all the Django 3 contributions into #50 to ensure a smoother merge process

**Comment by yakky** [comment]
@akinmetin thanks for your contribution. Django 3 support has been merged with #50

**Pull request closed without merging**